packages/fast-usdc/src/exos/settler.js

```diff
@@ -103,6 +105,11 @@ export const stateShape = harden({
 });
 
 /**
+ * Settler is responsible for monitoring (using receiveUpcall) deposits to the
+ * settlementAccount. It either "disburses" funds to the Pool (if funds were
+ * "advance"d to the payee), or "forwards" funds to the payee (if pool funds
+ * were not advanced).
```

Comment:
I think it's worth mentioning:

- it needs to register it's `tap` facet (`receiveUpcall`) via the `settlementAccount` so it gets events
- something about the `mintedEarly` flow - we don't `.forward()` until `notifier.checkMinted` is called by the `Advancer`.